encryption/BIKE/BIKE/impl/avx2.yaml

```diff
@@ -0,0 +1,7 @@
+name: AVX2
+platform: c
+hardware features:
+  - avx2
+type: optimized
+side channel guarding:
+  branching: true
```

Comment:
I'd say timing, too. They explicitly mention that memory accesses are protected, we could add this as a comment. (And maybe even as an option in the schema, what do you think?)

Reply:
I decided to mention this as a comment. See latest commit